### PR TITLE
[RW-5661][risk=no] Implement egress event audit page

### PR DESCRIPTION
**Pull request by calbach**
Stacked on https://github.com/all-of-us/workbench/pull/5977

image

### CODE REVIEW

File: ui/src/app/pages/admin/admin-egress-audit.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        {event.egressMegabytes?.toFixed(2)} MB (over {event.egressWindowSeconds / 60} min)
+      </DetailRow>
+      <DetailRow label="Status">
+        <Dropdown
+          value={pendingUpdateEvent?.status ?? event.status}
+          options={mutableEgressEventStatuses}
+          onChange={(e)=> {
+            setPendingUpdateEvent({
+              ...event,
+              status: e.value
```
Comment (yonghaoy) [inline ui/src/app/pages/admin/admin-egress-audit.tsx:115]: Shall we compare 
`egressDetails.status !== e.value?` here or in the button enabled below? 
i.e. we block user updating egress event if there is no diff

**Review by yonghaoy — APPROVED** [review]

File: ui/src/app/pages/admin/admin-egress-audit.tsx (hunk 1 above)
Comment (calbach) [reply ui/src/app/pages/admin/admin-egress-audit.tsx:116]: Good catch, added.

File: ui/src/app/pages/admin/admin-egress-audit.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      </DetailRow>
+      <DetailRow label="Sumologic event">
+        <textarea
+          readOnly
+          style={{width: '400px'}}
+          value={JSON.stringify(egressDetails.sumologicEvent, null, 2)} />
+      </DetailRow>
+    </div>
+    <FlexRow>
+      <Button
```
[the remark this answers is not in the document]
Comment (calbach) [reply ui/src/app/pages/admin/admin-egress-audit.tsx:129]: Basically it's just to discard any pending changes, which is common for an in-place edit UX. I changed this text to "discard changes" instead to make this more clear.

**Comment by calbach** [comment]
More failure from https://precisionmedicineinitiative.atlassian.net/browse/RW-7617, pulled into next sprint

**Pull request merged**